public/css/timeline.less

```diff
@@ -210,6 +260,25 @@
     }
   }
 
+  .btn-container {
+    background-color: mix(@gray-light, @body-bg-color, 25%);
+    margin-left: ~"calc(var(--sidebarWidth) * -1)";
```

Comment:
You added the negative margin here. To compensate for this you can add the same amount of padding-left and then the content will be centered.

`padding-left: ~"calc(var(--sidebarWidth))"; // short for ~"calc(var(--sidebarWidth) * -1) * -1"`

Reply:
Sorry, my fault. That suggestion doesn't make sense in the broader context. Please add the negativer margin only for `time-grid:not(.empty) and leave the padding-left out.